File: src/Punto8.java
import java.util.ArrayList;
import java.util.List;
import java.util.Random;

public static List<Integer> nuevalista() {
    List<Integer> lista = new ArrayList<>();
    Random random = new Random();
    for (int i = 0; i < 20; i++) {
        lista.add(random.nextInt(100) + 1);
    }
    return lista;
}

public void main() {
    List<Integer> lista = nuevalista();
    int Maximo = lista.stream()
            .mapToInt(v->v)
            .max()
            .orElse(0); // Si la lista esta vacía

    System.out.println(lista);
    System.out.println("Número con valor mas alto: " + Maximo);

    for (int i = 0; i < lista.size(); i++) {
        if (lista.get(i) == Maximo) {
            System.out.print("Indice del número: ");
            System.out.println(i+1);
        }
    }
}

File: Python/Punto_8.py
import random
n=int(input("Ingrese un tamaño para el vector: "))
lista=[0]*n
"""Se recorre cada posición del arreglo con map, cambiando el elemento
   por un número ingresado por usuario"""
resultado=list(map(lambda elemento:int(input("Ingrese un número: ")),lista))
# Se calculan el elemento mayor ingresado y su posicion en el vector
maximo=max(resultado)
posicion=resultado.index(maximo)
# Se imprimen la lista creada, su elemento mayor y su posición
print(resultado)
print(f"Máximo: {maximo}")
print(f"Posición del máximo: {posicion}")
